test/__snapshots__/Client.test.js.snap

```diff
@@ -0,0 +1,29 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Client console.log hot disabled 1`] = `
+Array [
+  "Hey.",
+  "[WDS] Live Reloading enabled.",
+]
+`;
+
+exports[`Client console.log hot enabled 1`] = `
+Array [
+  "[HMR] Waiting for update signal from WDS...",
+  "Hey.",
+  "[WDS] Hot Module Replacement enabled.",
+  "[WDS] Live Reloading enabled.",
+]
+`;
+
+exports[`Client console.log liveReload disabled 1`] = `
+Array [
+  "Hey.",
+]
+`;
+
+exports[`Client console.log liveReload enabled 1`] = `
+Array [
+  "Hey.",
```

Comment:
We should expect `[WDS] Live Reloading enabled.`

Reply:
Yes, need investigate what is problem